routing/routing_tests/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ include_directories(${OMIM_ROOT}/3party/osrm/osrm-backend/include)
 
 set(
   SRC
+  applying_traffic_test.cpp
```

Comment:
This one is the other way around: qmake files are fine and cmake files were stale.